### PR TITLE
Frames screen

### PR DESCRIPTION
**Pull request by LinnierGames**
### What's new
- FramesVc 

### What has changed
- Wrap Home Vc in a navigation controller 
- Wrap Add Frame Vc in a navigation controller 
- Present Add Frame Vc via Navigator

### CODE REVIEW

**Review by LinnierGames — COMMENTED** [review]
Self-reviewed

**Comment by LinnierGames** [comment]
[Create Frames Vc](https://trello.com/c/l5V6pjkC/57-create-frames-vc)

File: PicPi/HomeViewController.swift [hunk 1]
```diff
@@ -12,14 +12,18 @@ class HomeViewController: UIViewController     {
...
   let searchButton = customBtnRoundCornerBlueWithShadow(type: .custom)
   let addButton = UIButton(type: .custom)
+  let framesButton = UIButton(type: .system)
   var margins: UILayoutGuide!
 
   override func loadView() {
     super.loadView()
+
+    view.backgroundColor = .white
+    title = "PiPic"
```
Comment (RawiSoft) [inline PicPi/HomeViewController.swift]: you meant PicPi?

**Review by RawiSoft — APPROVED** [review]
Nice job!!

File: PicPi/HomeViewController.swift (hunk 1 above)
Comment (LinnierGames) [reply PicPi/HomeViewController.swift]: Nice catch! Done

**Pull request merged**